### PR TITLE
Fix #2507: Update NTP resource download timer.

### LINKED ISSUE #2507: Update NTP timer to try to fetch resources every hour
Old interval: 5 hours
New interval: 1 hour

### PR DESCRIPTION
**Pull request by iccub**
Now it tries to fetch from the server for every 1 hour.

## Summary of Changes

This pull request fixes #2507 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Comment by iccub** [comment]
Blocked to wait on final decision whether to include it for 1.16 release or later

**Review by kylehickinson — APPROVED** [review]
Approved even though we aren't merging yet

**Pull request merged**
